test/common.bash

```diff
@@ -167,21 +166,31 @@ pict_assert_equal () {
     ref=$1
     sample=$2
     pixel_max_ct=${3:-0}
-    sample_base=${sample%.*}
+    sample_base=$(basename "${sample%.*}")
     sample_ext=${sample##*.}
-    diff_=${sample_base}.diff.${sample_ext}
-    echo "reference:   ${ref}"
-    echo "sample:      ${sample}"
-    echo "diff image:  ${diff_}"
+    diff_dir=${BATS_TMPDIR}/"$(basename "$(dirname "$sample")")"
+    ref_bind="${ref}:/inp_a"
+    sample_bind="${sample}:/inp_b"
```

Comment:
These directory names seem kind of opaque?

Reply:
They are file names for the `.png` to be mounted over. Alternatives could be `bind_png_a` and `bind_png_b`.